.github/workflows/qa.yml

```
@@ -0,0 +1,150 @@
+
+    - name: Tag Docker image
+      run: |
+        docker tag $REPO_NAME:latest $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO_NAME:latest
+
+    - name: Push Docker image to ECR
+      run: |
+        docker push $AWS_ACCOUNT_ID.dkr.ecr.$AWS_REGION.amazonaws.com/$REPO_NAME:latest
+
+    - name: Send Slack notification on success
+      if: success()
+      run: |
```

actionlint (1.7.8)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)